inspire_json_merger/pre_filters.py

```diff
@@ -113,6 +117,29 @@ def filter_publisher_references(root, head, update):
     return root, head, update
 
 
+def update_authors_with_ordering_info(root, head, update):
+    """Adds ordering information into authors entries.
+
+    This is helpful so we won't loose correct ordering on conflicts.
+    Args:
+        root (pmap): the root record.
+        head (pmap): the head record.
+        update (pmap): the update record.
+
+    Returns: A tuple containing root, head and update with authors data
+        enriched with ordering information
+
+    """
+    if head.get('authors'):
```

Comment:
```suggestion
    if 'authors' in head:
```
the schema guarantees that if it's present, it's a non-empty list.

Reply:
:+1: